### PR TITLE
Ktl 490 new visual style why teach page

### PR DESCRIPTION
**Pull request by krutilov**
https://branch-ktl-490-new-visual-style-why-teach-page.kotlin-web-site.labs.jb.gg/education/why-teach-kotlin.html

### CODE REVIEW

File: static/js/ktl-component/why-teach/index.jsx [hunk 1]
```diff
@@ -0,0 +1,663 @@
...
+                      <li>
+                        — Kotlin has the fastest growing language community. (<a className="rs-link"
+                                                                                 href="https://developer-economics.cdn.prismic.io/developer-economics/dbf9f36f-a31a-440a-9c22-c599cc235fa4_20th+edition+-+State+of+the+developer+Nation.pdf">SlashData's
+                        State of the Developer Nation 20th
+                        edition, Q1 2021</a>)
+                      </li>
+                    </ul>
+                  </p>
+
+                  <QuotesSlider quotes={[
```
Comment (zoobestik) [inline static/js/ktl-component/why-teach/index.jsx:115]: Top module constant for complex object is better for time and potentially better for render optimization... But inline is really better for reading. 🤔

@nikpachoo, @krutilov,, what do you think about use constant complex objects props?

File: static/js/ktl-component/why-teach/style.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      max-width: 70%;
+    }
+    &__info {
+      display: block;
+      margin-left: 32px;
+    }
+  }
+}
+
+.ktl-with-anchor {
```
Comment (zoobestik) [inline static/js/ktl-component/why-teach/style.scss:40]: What does `ktl-with-anchor` do? if it's should have anchor link, i haven't seen it.

Comment (zoobestik) [reply static/js/ktl-component/why-teach/style.scss:40]: Actually it is a good case to show link id as on main. May be we will create task for that i component?

File: static/js/ktl-component/why-teach/index.jsx [hunk 3]
```diff
@@ -0,0 +1,663 @@
...
+                Language of the industry
+              </h3>
+
+              <div className="quote-section__grid">
+                <div className="quote-section__content">
+
+                  <p className="ktl-text-1 ktl-offset-bottom-m">
+                    <ul>
+                      <li className="ktl-offset-bottom-s">
+                        — Kotlin is used by top companies such as Google, Twitter, Reddit, Netflix, Uber, BMW, Coursera,
```
Comment (zoobestik) [inline static/js/ktl-component/why-teach/index.jsx:89]: Actually, the good practise is use bullets outside. Can we use [Typography/Lists](https://rescui.dev/#/Typography/Lists)? Designer agree with that.

**Review by zoobestik — CHANGES_REQUESTED** [review]

File: static/js/ktl-component/why-teach/index.jsx (hunk 1 above)
Comment (krutilov) [reply static/js/ktl-component/why-teach/index.jsx:115]: My initial thought was to move data into object, but as you mention readability is much better with inline data, so i stick with this approach

File: static/js/ktl-component/why-teach/style.scss (hunk 2 above)
Comment (krutilov) [reply static/js/ktl-component/why-teach/style.scss:40]: It's compensation for sticky header

File: static/js/ktl-component/why-teach/index.jsx (hunk 3 above)
Comment (krutilov) [reply static/js/ktl-component/why-teach/index.jsx:89]: Agree, will fix

File: templates/pages/education/why-teach-kotlin.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    {% ktl_component "why-teach" path=request.path %}
+{% endblock %}
+
+{% block styles %}
+    <link rel="stylesheet" href="{{ url_for('static', filename='education.css')|autoversion }}">
+{% endblock %}
+
+{% block scripts %}
+    <script src="{{ url_for('static', filename='education.js')|autoversion }}"></script>
+{% endblock %}
```
[the remark this answers is not in the document]
Comment (krutilov) [reply templates/pages/education/why-teach-kotlin.html]: Nope, removed

File: static/js/ktl-component/why-teach/index.jsx (hunk 1 above)
Comment (nikpachoo) [reply static/js/ktl-component/why-teach/index.jsx:115]: For me is preferable to separate data from template:
- easy to read template without bunch of data
- template changes will not affect the data (for example remove or add extra parent div will affect data indent too)

Comment (krutilov) [reply static/js/ktl-component/why-teach/index.jsx]: Moved to json

**Review by nikpachoo — APPROVED** [review]

File: static/js/ktl-component/teach/style.scss [hunk 5]
```diff
@@ -124,6 +126,7 @@
 
   &__bottom {
     display: none;
+
     br {
```
Comment (zoobestik) [inline static/js/ktl-component/teach/style.scss:130]: 🥲

**Review by zoobestik — APPROVED** [review]

**Pull request merged**